akka-http-core/src/main/scala/akka/http/scaladsl/model/Multipart.scala

```diff
@@ -43,20 +53,48 @@ sealed trait Multipart {
         .flatMapConcat(ConstantFun.scalaIdentityFunction)
     HttpEntity.Chunked(mediaType withBoundary boundary withCharset charset, chunks)
   }
+
+  /** Java API */
```

Comment:
Oh, right, I forgot about the slightly baroque way the Scala and Java APIs inherit each other in HTTP, please disregard the removal part. But who will add ScalaDoc to all the other methods?